web/src/components/AppBar/UserMenu.vue

```diff
@@ -47,12 +47,19 @@ export default {
     user,
     userInitials() {
       if (this.user) {
+        // TODO Girder code, delete
         const { firstName, lastName } = this.user;
         if (firstName && lastName) {
           return (
             firstName.charAt(0).toLocaleUpperCase() + lastName.charAt(0).toLocaleUpperCase()
           );
         }
+        const { first_name, last_name } = this.user;
+        if (first_name && last_name) {
+          return (
+            first_name.charAt(0).toLocaleUpperCase() + last_name.charAt(0).toLocaleUpperCase()
+          );
+        }
```

Comment:
I think if we're going to implicitly check if the Django toggle is on, we should make the comment here more clear. Also, adding a comment above the second block would be nice too